localenv/README.md

```diff
@@ -175,7 +174,10 @@ The Examples folder in the Bruno collection includes an [Open Payments](https://
 5. requests a grant to create (and read) a quote on the sender's account
 6. creates a quote on the sender's account
 7. requests a grant to create (and read) an outgoing payment on the sender's account
-8. continues the grant request (via the interaction flow)
+
+Note that you have to go through an interaction flow by clicking on the `redirect` link in the grant request response. Cope the `interaction_ref` from the url query parameters after clicking "Continue" in the interaction flow.
```

Comment:
I added this because polling does not work yet, I think. The spec has not been updated so the request validation throws when the continue request does not include the `interaction_ref`. I'm in the process of confirming with @njlie

Reply:
I think "Cope" should be copy :)

Reply:
@sabineschaller Oddly enough it seemed to work for us in Postman (although no one knows why and apparently no longer works for Max) but not Bruno. I think it was testd originally against Postman so we thought it was working but I just ran into this myself and made an issue to update the spec https://github.com/interledger/open-payments/issues/433